config/default/AuthorizationServerFactory.conf.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+use oat\taoLti\models\classes\Security\AuthorizationServer\AuthorizationServerFactory;
+
+return new AuthorizationServerFactory(
+    [
+        AuthorizationServerFactory::OPTION_ENCRYPTION_KEY => 'verySecretKeyToReplace',
```

Comment:
How will we manage the storage of this value?

Reply:
Can we just generate this in a migration then?
https://github.com/oat-sa/extension-tao-lti/pull/268/commits/ab6ee2c4289c14500c9589e66011e63a2d8be1df

Reply:
It should be in some storage, if we lost the configurations, we lost the key. Also this is an `OAUTH2_ENCRYPTION_KEY`. What about give more descriptive name in the const?

Reply:
like in Rdf? Why we may lost configuration? Isn't that already storage?

Reply:
@mike-ionut-mihai-sandu-tao @boajer can you provide your opinion about this?

Reply:
not a blocker for the task , we can leave it like that and improve it in the next task